saleor/core/exceptions.py

```diff
@@ -1,4 +1,8 @@
+from ..checkout.error_codes import CheckoutErrorCode
```

Comment:
Core module should not take any dependencies from other Saleor's module.

Reply:
Ideally yes; in this case, the exception is either used in `product` / `checkout` or `order` modules. Maybe instead of `core`, it should be defined inside one of them (not sure which one though). We will anyway end up importing it between those modules. @korycins @k-brk  If you have better idea then we can move it, if not I would leave it here.

Reply:
I've created a ticket for that: #4744 
We can decide it later.